interfaces/story-dao-I.ts

```diff
@@ -2,13 +2,14 @@
  * @file Declares API for Stories related data access object methods
  */
 import Story from "../models/stories/story";
+import User from "../models/users/user";
 
 export default interface StoryDaoI {
   findStories(): Promise<Story[]>;
   findStoryById (sid: string): Promise<any>;
   findStoriesByUser (uid: string): Promise<Story[]>;
-  findStoriesByVisibility (visibility: string): Promise<Story[]>;
-  createStory (uid: string, story: Story): Promise<Story>;
+  findStoriesVisibleToUser (uid: string): Promise<Story[]>;
+  createStory (uid: string, viewers: string[], story: Story): Promise<Story>;
```

Comment:
`viewers` array will be present in `Story` object right?

Reply:
+